supabase/authentication/email_templates/confirm_signup.html

```diff
@@ -0,0 +1,50 @@
+<!-- Welcome to Motis Ride - Confirm Your Signup -->
+
+<table style="width:100%;border-collapse:collapse;border:0;border-spacing:0;background:#ffffff;" role="presentation">
+  <tbody><tr>
+    <td style="padding:0;" align="center">
+      <table style="width:602px;border-collapse:collapse;border:1px solid #cccccc;border-spacing:0;text-align:left;" role="presentation">
+        <tbody><tr>
+          <td style="padding:15 0 15 0;background:#70bbd9;" align="center">
+              <img style="height:auto;display:block;" width="100" src="https://khvzghkhzkjmnbagbzkm.supabase.co/storage/v1/object/public/public/logo.png">
```

Comment:
I have uploaded the logo into a "public" bucket (inside the public namespace, that's why the url looks so weird). Seems to work well for hosting images.

Now we just need to find better images (and better text) for the rest of this mail.